db_test.go

```diff
@@ -425,6 +426,52 @@ func (s *SurrealDBTestSuite) TestSmartMarshalQuery() {
 	})
 }
 
+func (s *SurrealDBTestSuite) TestConcurrent() {
```

Comment:
we can call this TestConcurrentOperations

Reply:
done